tests/unit/test_config.py

```diff
@@ -28,13 +28,74 @@ def test_config_good_dict(good_config_dict):
     assert Config.from_config(config_dict=good_config_dict) == good_config_dict
 
 
-def test_config_missing_values(config_defaults):
+def test_config_extra_top_level_value(good_config_dict):
     """
-    GIVEN a json file containing an incomplete definition of a Bugjira config
+    GIVEN a dict containing a Bugjira config with an extra bugzilla setting
```

Comment:
Should this just say "extra setting" here, since it's not under the bugzilla category unlike the test below?